### DISCUSSION TITLE
How to force training on non shuffle dataloader?

### DISCUSSION THREAD

**Discussion by ZENGYIMING-EAMON** [Lightning-AI/pytorch-lightning#13121] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
The PL will force turning on shuffle=True when training but when I want to overfit the whole training set I want it to be shuffle=False.
